src/types/quizzes.d.ts

```diff
@@ -0,0 +1,71 @@
+import { ConstructorClientOptions, NetworkParameters } from '.';
+
+export default Quizzes;
+
+export interface QuizzesParameters {
+  section?: string;
+  answers?: any[];
+  versionId?: string;
+}
+
+declare class Quizzes {
+  constructor(options: ConstructorClientOptions);
+
+  options: ConstructorClientOptions;
+
+  getQuizNextQuestion(
+    id: string,
+    parameters?: QuizzesParameters,
+    networkParameters?: NetworkParameters
+  ): Promise<NextQuestionResponse>;
+
+  getQuizResults(
+    id: string,
+    parameters?: QuizzesParameters,
+    networkParameters?: NetworkParameters
+  ): Promise<QuizResultsResponse>;
+}
+
+/* quizzes results returned from server */
+export interface NextQuestionResponse extends Record<string, any> {
+  next_question: Partial<NextQuestion>;
+  is_last_question?: boolean;
+  version_id?: string;
+}
+export interface QuizResultsResponse extends Record<string, any> {
+  result: Partial<QuizResult>;
+  version_id?: string;
+}
+
+export interface NextQuestion extends Record<string, any> {
+  id: number;
+  title: string;
+  description: string;
+  type: 'single' | 'multiple' | 'open' | 'cover';
+  cta_text: string;
+  images: Partial<QuestionImages>;
+  options: Partial<QuestionOption>[];
```

Comment:
Should this be partial? I think a couple of the attributes are required inside QuestionOption